src/portal/lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@harbor/ui",
-    "version": "1.0.0-rc7",
+    "version": "1.7.1-toggle-retag",
```

Comment:
I don't think `1.7.1-toggle-retag` is a reasonable tag, please rename it to be `1.7.1-pacth1`

Reply:
resovled